packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -23,11 +23,11 @@ export interface ICalendarState {
 
 export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> implements ICalendar {
   public static defaultProps: ICalendarProps = {
-    onSelectDate: null,
-    onDismiss: null,
+    onSelectDate: undefined,
+    onDismiss: undefined,
     isMonthPickerVisible: true,
     isDayPickerVisible: true,
-    value: null,
+    value: undefined,
```

Comment:
Most of these conversions from null to undefined seemed safe to do (and avoid the need to update prop/state signatures to include '|null'), but holler if you see something broken.